keyboards/dz60/keymaps/hailbreno/config.h

```diff
@@ -0,0 +1,54 @@
+#ifndef CONFIG_H
```

Comment:
This file should only be: 
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#endif
```

You're not changing any of the other info, so you don't need to define it here.  It's redundant